### PR TITLE
Added missing release note detail for MB-32497

### PR DESCRIPTION
**Pull request by RayOffiah**
Added release note for https://issues.couchbase.com/browse/MB-32497

Request: https://issues.couchbase.com/browse/DOC-9377

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -546,6 +546,8 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | https://issues.couchbase.com/browse/MB-32685[MB-32685^]
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
+| https://issues.couchbase.com/browse/MB-32497[MB-32497^]
+| *Summary*: Connection::signalIfIdle may cause a deadlock when used from a different thread
```
Comment (rdemellow) [inline modules/release-notes/pages/relnotes.adoc]: Think we need to add more explanation to what the user might see e.g.
"Memcached connections may hang due to Connection::signalIfIdle triggering a deadlock when used from a different thread" Think adding the words connection and hang will help to make it easier for people when searching though the release notes :)

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -546,6 +546,8 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | https://issues.couchbase.com/browse/MB-32685[MB-32685^]
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
+| https://issues.couchbase.com/browse/MB-32497[MB-32497^]
+| *Summary*: Memcached connections may hang due to `Connection::signalIfIdle` triggering a deadlock when used from a different thread
```
Comment (rdemellow) [inline modules/release-notes/pages/relnotes.adoc]: Missing a full stop?

**Review by rdemellow — CHANGES_REQUESTED** [review]

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (RayOffiah) [reply modules/release-notes/pages/relnotes.adoc]: Ouch!

**Review by rdemellow — APPROVED** [review]

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (osfameron) [inline modules/release-notes/pages/relnotes.adoc]: I've not worked on server release notes, but this looks more like the initial problem statement?
Would it be better to reword as what has *changed* (e.g. the improvement made)

(Please ignore if I've missed the point/convention for server!)

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -547,7 +547,7 @@ WARNING: Changing the internal replication setting will cause all XDCR processes
 | *Summary*: Introduced a delay in updating the RBAC database revision number to avoid returning AUTH_STALE during a refresh of the RBAC database.
 
 | https://issues.couchbase.com/browse/MB-32497[MB-32497^]
-| *Summary*: Connection::signalIfIdle may cause a deadlock when used from a different thread
+| *Summary*: Memcached connections may hang due to `Connection::signalIfIdle` triggering a deadlock when used from a different thread
```
Comment (osfameron) [inline modules/release-notes/pages/relnotes.adoc]: Ah sorry, I missed this rewording: this is better, but I think it's still under the fixed issues section? So better worded as what has been fixed?

**Review by osfameron — APPROVED** [review]
This looks OK, minor niggle on the release note.

**Pull request merged**